courses/free-courses-ar.md

```diff
@@ -381,7 +381,8 @@
 * [دورة انشاء برنامج فواتير php- laravel&rlm;](https://www.youtube.com/playlist?list=PLftLUHfDSiZ7pKXkpGCoZATm5rF6msj5A) - Mora Soft
 * [دورة Laravel 6/7/8 PHP&rlm;](https://www.youtube.com/playlist?list=PLMYF6NkLrdN8V2JKIMxqMsZNPsgUj3WOK) - Muhammed Essa
 * [كورس اساسيات لارافيل 9](https://www.youtube.com/playlist?list=PLftLUHfDSiZ4GfPZxaFDsA7ejUzD7SpWa) - Mora Soft
-* [مشروع لارافل متجر الكتروني متعدد التجار واللغات متكامل - تطبيقات لارافل عمليه](https://www.youtube.com/playlist?list=PLCm7ZeRfGSP7DCb-NamG2tt7uQUfxP2va) - Professional Code
+* [لارافيل للمبتدئين](https://www.youtube.com/playlist?list=PLWCBAKY7-4buQazvDjeZhjLl54UqbF3lM) - Ahmed Abd El Ftah
+*  [مشروع لارافل متجر الكتروني متعدد التجار واللغات متكامل - تطبيقات لارافل عمليه](https://www.youtube.com/playlist?list=PLCm7ZeRfGSP7DCb-NamG2tt7uQUfxP2va) - Professional Code
```

Comment:
is there an extra space here?

Reply:
i don't think so.

Reply:
there are 2 spaces after `*` in line 385, all the other entries have just on e space

Reply:
Yes I solved the problem.